GenderPayGap.WebUI/Views/ManageOrganisations/UsersRegisteredToReportForOrganisation.cshtml

```diff
@@ -1,35 +1,130 @@
 ﻿@using GenderPayGap.Database
+@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
 @model GenderPayGap.WebUI.Models.ManageOrganisations.ManageOrganisationViewModel
 
 @{
     var encryptedOrganisationId = Encryption.EncryptQuerystring(Model.Organisation.OrganisationId.ToString());
-}
 
-<table class="govuk-table">
-    <thead class="govuk-table__head">
-        <tr class="govuk-table__row">
-            <th scope="col" class="govuk-table__header">User</th>
-            <th scope="col" class="govuk-table__header">Action</th>
-        </tr>
-    </thead>
-    <tbody class="govuk-table__body">
-        @foreach (User userRegisteredToReport in Model.GetFullyRegisteredUsersForOrganisationWithCurrentUserFirst())
+    var desktopHeadings = new List<TableCellViewModel>
+    {
+        new TableCellViewModel
+        {
+            Text = "User",
+            Colspan = 1
+        },
+        new TableCellViewModel
         {
-            var encryptedUserId = Encryption.EncryptQuerystring(userRegisteredToReport.UserId.ToString());
-            <tr class="govuk-table__row">
-                <td class="govuk-table__cell">
-                    @userRegisteredToReport.Fullname
-                    @if (userRegisteredToReport.UserId == Model.User.UserId)
-                    {
-                        <span>(You)</span>
-                    }
-                </td>
-                <td class="govuk-table__cell">
-                    <a class="govuk-link" href="@(Url.Action("RemoveOrganisation", "Organisation", new {orgId = encryptedOrganisationId, userId = encryptedUserId}))">
-                        Remove user <span class="govuk-visually-hidden">@userRegisteredToReport.Fullname</span>
-                    </a>
-                </td>
-            </tr>
+            Text = "Action",
+            Colspan = 1
         }
-    </tbody>
-</table>
+    };
+    var mobileRows = new List<TableRowViewModel>();
+    @foreach (User userRegisteredToReport in Model.GetFullyRegisteredUsersForOrganisationWithCurrentUserFirst())
+    {
+        var encryptedUserId = Encryption.EncryptQuerystring(userRegisteredToReport.UserId.ToString());
+        var newRow = new TableRowViewModel
+        {
+            Row = new List<TableCellViewModel>
+            {
+                new TableCellViewModel
+                {
+                    Html = @<text>
+                               <p class="govuk-!-margin-top-1 govuk-!-margin-bottom-1">
+                                   @userRegisteredToReport.Fullname
+                                   @if (userRegisteredToReport.UserId == Model.User.UserId)
+                                   {
+                                       <span>(You)</span>
+                                   }

```

Comment:
Feels like there must be some de-duplication we can do with bits of html like this. It's definitely not worth making a whole partial but a private function so that the desktop and mobile code share as much as possible might be worth it. What do you think?